src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1234,7 +1324,6 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		case SN_ConvertToUInt64:
 		case SN_Create:
 		case SN_Dot:
-		case SN_ExtractMostSignificantBits:
```

Comment:
@fanyang-mono is this now the right way to enable implemented intrinsic?

Reply:
Yes.